Mathlib/Probability/Distributions/Gaussian.lean

```diff
@@ -6,6 +6,7 @@ Authors: Lorenzo Luccioli, Rémy Degenne, Alexander Bentkamp
 import Mathlib.Analysis.SpecialFunctions.Gaussian.GaussianIntegral
 import Mathlib.Probability.Notation
 import Mathlib.MeasureTheory.Decomposition.Lebesgue
+import Mathlib.Probability.Moments
```

Comment:
Think about writing `#min_imports` at the bottom of your file from time to time
```suggestion
import Mathlib.Analysis.SpecialFunctions.Gaussian.GaussianIntegral
import Mathlib.Probability.Moments
```

Reply:
Did not know about this command, thanks!